### ISSUE TITLE
Multi icon creations 

### ISSUE THREAD

**Issue by justBytemc** [DavidBerdik/Let-Me-Downgrade#10]
Actually, the question in the topic is why a new icon is created after a period of time.... and you can hide the icon, even if there are 100 of them, but I haven’t seen them.
![IMG_20241027_070522](https://github.com/user-attachments/assets/64de4354-ae29-435e-be95-c1a958af13cb)

**Comment by DavidBerdik** [comment]
This is a very unusual issue and certainly something that I've never seen before! Are these icons being placed on your home screen?

**Comment by justBytemc** [comment]
Yes, on the screen and so that they don’t get in the way I throw them into a folder.

**Comment by DavidBerdik** [comment]
Could you please try removing "System UI" from the module scope in the LSPosed Manager, rebooting your phone, and checking to see if the issue continues to occur?

**Comment by justBytemc** [comment]
Sure, but now I've reinstalled and I have only one icon for now, I'll disable systemui and see if more appear. Reinstalling once a week is also not a very interesting activity...

**Comment by DavidBerdik** [comment]
> Sure, but now I've reinstalled and I have only one icon for now, I'll disable systemui and see if more appear.

I'm hoping it does, but either way, please let me know of the outcome.

> Reinstalling once a week is also not a very interesting activity...

Yeah I totally understand that. Unfortunately, if removing System UI from the scope does not fix the issue, I am not sure if I will be able to properly fix the issue for you since whatever is going on seems to be specific to the Android distribution from your OEM. Or at least, I haven't received reports of this issue from anyone else, which leads me to believe that whatever's going on here is unique to your type of phone.

**Comment by justBytemc** [comment]
Heh,it say hello again.![Screenshot_20241030_151704](https://github.com/user-attachments/assets/00237ab1-4c07-4aee-ba1b-0924f6ffc9f2)

**Comment by DavidBerdik** [comment]
Would you be willing to try installing this modified version of the app on your phone and seeing if the problem persists? I put the APK inside a zip file because GitHub does not allow APKs to be attached directly to comments on issue threads. Also, please keep in mind that although I don't expect this build to have any issues, I did not properly test it due to lack of time.

[app-debug.zip](https://github.com/user-attachments/files/17582217/app-debug.zip)

**Comment by justBytemc** [comment]
Okay, no problem. Installed, let's wait. Could you make a function to hide the icon?

**Comment by DavidBerdik** [comment]
Great! 

Let Me Downgrade used to have the functionality to hide the icon but I removed it because Google's awful automation kept flagging the app as malicious when I tried to publish updates. If you want to hide the icon, I would recommend using [this](https://github.com/KieronQuinn/PixelLauncherMods).

**Comment by DavidBerdik** [comment]
@justBytemc Hello! Are you still encountering this issue?

**Comment by justBytemc** [comment]
Hi, since I installed the version with the fix, there has been no problem. Thank you.

**Comment by DavidBerdik** [comment]
Great! In that case, I will include this fix in the next stable release. Thank you!

**Comment by DavidBerdik** [comment]
The changes that were tested in the experimental build that I provided to @justBytemc are included in the release of v1.0.6, so I am going to close this now.
